sensor/common/component.go

```diff
@@ -26,6 +28,21 @@ type SensorComponent interface {
 	ResponsesC() <-chan *central.MsgFromSensor
 }
 
+// MessageToComplianceWithAddress adds the Hostname to sensor.MsgToCompliance so we know where to send it to.
+type MessageToComplianceWithAddress struct {
+	Msg       *sensor.MsgToCompliance
+	Hostname  string
+	Broadcast bool
+}
+
+// ComplianceComponent is responsible for handling arriving NodeInventory messages, processing them, and sending them to central
```

Comment:
ComplianceComponent is a sensor component that can communicate with compliance. All the messaged intended for compliance are returned by ComplianceC(). It _must_ be started before the compliance.Multiplexer or we panic.

Reply:
https://issues.redhat.com/browse/ROX-16931